src/schema/contentful/phoenix.js

```diff
@@ -76,6 +78,24 @@ const typeDefs = gql`
     ${entryFields}
   }
 
+  type PersonBlock implements Block {
+    "Name of the person displayed on the block"
+    name: String
```

Comment:
This is a required field
```suggestion
    name: String!
```